spec/compiler/crystal/tools/context_spec.cr

```diff
@@ -1,26 +1,24 @@
 require "spec"
 require "yaml"
-require "../../../../src/compiler/crystal/**"
```

Comment:
@RX14 My comment applies when you do `$ crystal spec spec/compiler/.../`. i.e. no `bin/crystal` wrapper is called.

At least for a DRY about relative vs crystal path we should change this to require spec_helper. But again that is not about adding private to methods and classes.